src/store/helpers.ts

```diff
@@ -1,6 +1,35 @@
 import type { Commit, Dispatch } from 'vuex'
 import type { RootState } from './types'
 import { SocketActions } from '@/api/socketActions'
+import type { AppPushNotification } from './notifications/types'
+
+export const handleTrinamicDriversChange = (payload: any, state: RootState, dispatch: Dispatch) => {
+  for (const item in payload) {
+    const [type, nameFromSplit] = item.split(' ', 2)
+
+    if (
+      /^tmc\d{4}$/.exec(type) &&
+      payload[item]?.drv_status?.otpw != null &&
+      state.printer.printer?.[item]?.drv_status?.otpw == null
+    ) {
+      const name = nameFromSplit ?? item
+
+      const notification: AppPushNotification = {
+        id: `${item}-otpw`,
+        title: `TMC '${name}' is over-heating`,
```

Comment:
I feel like this should be something like `Stepper driver 'name' is over-heating`, chances are end users don't know what TMC means

Reply:
Good point, I updated the description to your suggestion!